templates/include/telemetry/gcItem.1.schema.json

```diff
@@ -11,7 +11,7 @@
         "properties": {
           "times": {
             "type": "object",
-            "maxProperties": 65
+            "maxProperties": 72
```

Comment:
Why 72 instead of 73 as referenced in the bug?

Reply:
Oops.

There are 73 phases, numbered 0 to 72.  I'm not sure if phase 0 is used but lets make this 73 and I'll fix the mozilla-central patch and also document it there.

Thanks for catching this.